docs/defects-api.yml

```diff
@@ -127,14 +127,14 @@ paths:
 
 components:
   #securitySchemes:
-    #OAuth2:
-      #type: oauth2
-      #flows:
-        #authorizationCode:
-          #authorizationUrl: "https://example.com/oauth/authorize"
-          #tokenUrl: "https://example.com/oauth/token"
-          #scopes:
-            #read: Grants read access
+  #OAuth2:
```

Comment:
Appreciate these are commented out at the moment, but should we leave the formatting as it was for when it does get commented back in.